superduper/base/datalayer.py

```diff
@@ -180,58 +162,8 @@ def initialize_vector_searcher(
         vector_comparison = vector_search_cls.from_component(vi)
 
         assert isinstance(clt.identifier, str), 'clt.identifier must be a string'
-
-        self.backfill_vector_search(vi, vector_comparison)
-
         return FastVectorSearcher(self, vector_comparison, vi.identifier)
 
-    def backfill_vector_search(self, vi, searcher):
```

Comment:
There is no use of backfill since on `schedule_jobs` of `vector_index` we are scheduling copy vector jobs
These copy vector jobs update the vector search 

This process is basically backfill (with schedule_jobs)
on
`db.apply(vector_index)`

Reply:
Good idea.